core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -203,8 +205,11 @@ angular.module('oppia').component('explorationEditorPage', {
         UserEmailPreferencesService, UserExplorationPermissionsService,
         UserService, WindowDimensionsService) {
       var ctrl = this;
+      var reconnectedMessageTimeoutMilliseconds = 4000;
+      var disconnectedMessageTimeoutMilliseconds = 5000;
       ctrl.directiveSubscriptions = new Subscription();
       ctrl.autosaveIsInProgress = false;
+      ctrl.connectionService = ConnectionService;
```

Comment:
Ditto*

Reply:
Removed it as it was needed before but not now.